README.md

```diff
@@ -1,3 +1,45 @@
-# resin.io docs
+Resin Documentation
+---------
 
-This repo contains our public-facing documentation.
+[![npm version](https://badge.fury.io/js/docs.svg)](http://badge.fury.io/js/docs)
+[![dependencies](https://david-dm.org/resin-io/docs.png)](https://david-dm.org/resin-io/docs.png)
+[![Build Status](https://travis-ci.org/resin-io/docs.svg?branch=master)](https://travis-ci.org/resin-io/docs)
+[![Build status](https://ci.appveyor.com/api/projects/status/qbsivehgnq0vyrrb?svg=true)](https://ci.appveyor.com/project/fedealconada/resin-docs)
+
+Join our online chat at [![Gitter chat](https://badges.gitter.im/resin-io/chat.png)](https://gitter.im/resin-io/chat)
+
+The official JavaScript [Resin.io](https://resin.io/) Documentation.
+
+Role
+----
+
+The intention of this repo is to provide our public-facing documentation.
+
+
+Installation
+------------
+
+Install the Resin Documentation by running:
```

Comment:
Probably better to rephrase as "To run the docs web application locally do..." or something like that. Maybe even worth to rename the "Installation" header to "Running locally"?